### PR TITLE
Configure the Cache-Control header for fonts

### PR DESCRIPTION
**Pull request by roj1512**

### CODE REVIEW

**Comment by roj1512** [comment]
Works so smooth. It feels like the font is locally installed.

image

**Comment by roj1512** [comment]
Perhaps we can also add Cache-Control to other assets as well so we feel even smoother.

File: site/netlify.toml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[[headers]]
+  for = "/fonts/*"
+  [headers.values]
+  cache-control = "max-age=604800"
```
Comment (KnorpelSenf) [inline site/netlify.toml:4]:
```suggestion
  cache-control = "max-age=604800" # 1 week
```

**Review by KnorpelSenf — APPROVED** [review]
LGTM!

**Pull request merged**